content/doc/administration/requirements/linux.adoc

```diff
@@ -0,0 +1,78 @@
+---
+layout: documentation
+title:  Linux Support Policy
+---
+
+This page documents the Linux support policy for the Jenkins controller and agents.
+
+== Scope
+
+Jenkins plugins may set additional requirements to Linux versions on controllers and/or agents.
+This page does not document such requirements.
+Please refer to plugin documentation.
+
+== Why?
+
+Theoretically, Jenkins can run everywhere where you can run a supported Java version,
+but there are some limitations in practice.
+Jenkins core and some plugins include native code or depend on Linux API and subsystems,
+and hence they rely on specific Linux versions.
+Jenkins platform specific installation packages rely on specific Linux versions.
+
+== Support levels
+
+We define multiple support levels for Linux platforms.
+
+[width="100%",cols="20%,35%,45%",options="header",]
+|===
+|Support level |Description |Platforms
+
+| **Level 1** - Full support
+| We run automated testing for these platforms, and we intend to fix the reported issues timely.
+a|
+  * 64-bit (amd64) Linux versions supported by Debian, Red Hat, SUSE, and Ubuntu
+  * Linux versions used in the official Docker images
+
+| **Level 2** - Supported
+| We do not actively test these platforms, but we intend to keep compatibility.
+  We will consider patches if they do not put Level 1 support at risk and do not create maintenance overhead.
+a|
+  * 64-bit (amd64, arm64, s390x, ppc64le) Linux versions that use the Debian packaging format
+  * 64-bit (amd64, arm64, s390x, ppc64le) Linux versions that use the rpm packaging format
+
+| **Level 3** - Patches considered
+| Support may have limitations and extra requirements.
+  We do not test compatibility, and we may drop support if there is a need.
```

Comment:
```suggestion
  We do not test compatibility, and we may drop support at any time.
```
I thought about "we may drop support when necessary" but isn't the point that we can drop support at any time?